### PR TITLE
Update a.rb

### PR DESCRIPTION
**Pull request by tankxu**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/tankxu/test-ci/pull/1%23issuecomment-277608120%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/1%23discussion_r99531742%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/1%23issuecomment-277608120%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22adas%22%2C%20%22created_at%22%3A%20%222017-02-06T07%3A55%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1388582%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/vangie%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%203c3f2bfdff4aa533241182cfc3f16c15783ca653%20a.rb%203%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/1%23discussion_r99531742%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22jhjkkk%22%2C%20%22created_at%22%3A%20%222017-02-06T08%3A02%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20a.rb%3AL1-4%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- <a href='https://github.com/vangie'></a> adas
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/1#issuecomment-277608120'>General Comment</a></b>
- [ ] <a href='#crh-comment-Pull 3c3f2bfdff4aa533241182cfc3f16c15783ca653 a.rb 3'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/1#discussion_r99531742'>File: a.rb:L1-4</a></b>
- <a href='https://github.com/tankxu'></a> jhjkkk

<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/tankxu/test-ci/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by vangie** [comment]
adas

File: a.rb [hunk 1]
```diff
@@ -1 +1,3 @@
 puts "hello"
+jjj
+gg
```
Comment (tankxu) [inline a.rb:3]: jhjkkk

Comment (tankxu) [reply a.rb:3]: asdf

**Review by tankxu — COMMENTED** [review]
asdf

**Review by fatjyc — APPROVED** [review]

**Review by summershrimp — APPROVED** [review]
hahah

**Review by summershrimp — DISMISSED** [review]
need change

**Pull request closed without merging**